generators/client/files-react.js

```diff
@@ -313,7 +313,6 @@ const files = {
             path: TEST_SRC_DIR,
             templates: [
                 'jest.conf.js',
-                'spec/enzyme-setup.ts',
```

Comment:
Cleanup this file.

Reply:
I deleted this file

Reply:
Remove the file at:
https://github.com/jhipster/generator-jhipster/blob/03980190ab493f251166841a5bc1741e9b95af17/generators/cleanup.js#L130

Reply:
yes done